debian/update.sh

```diff
@@ -45,22 +45,27 @@ if [ -n "$PIKERNELMODDIR" ] ; then
     fi
 fi
 
+HOST_ARCH="$(uname -m)"
 ARCH=${ARCH:-arm}
 case "$ARCH" in
-        arm)
```

Comment:
Why is the indention changing here?

Reply:
Even if I don't like this pattern (though it's one of the two most used ones): Code format tool (shfmt). From the point of view of reproducibility and consistency, it's a bullet I'm willing to take.